example/main.go

```diff
@@ -60,11 +61,11 @@ func main() {
 
 	result, err := svc.CreateMultipartUpload(input)
 	if err != nil {
-		log.ErrorC("error creating mpu", err, nil)
+		log.Event(context.TODO(), "error creating mpu", log.ERROR)
```

Comment:
```suggestion
		log.Error(context.TODO(), "error creating mpu", err)
```

Reply:
I get "cannot use err (variable of type error) as "github.com/ONSdigital/log.go/v2/log".severity value in argument to log.Event" if I implement this suggestion.